examples/dataflow-cloud-run/Dockerfile

```diff
@@ -0,0 +1,42 @@
+#    Copyright 2019 Google LLC
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
```

Comment:
This seems inline with the vision for  go/dataflow-dynamic-templates @sabhyankar is there a docker base image the dataflow team is curating for activities like this?

Reply:
nit: To make this consistent with the vision for dynamic templates I think we should use 
`gcr.io/dataflow-templates-base/java8-template-launcher-base` base image (which is maintained by google.

Eventually i see all cloud run / DF template use cases being replaced by dynamic templates and I think using this base image for this example would ease migration in the future.

Reply:
I dont see any issue using that flex template base except that there is a pre-configured entrypoint in those images. I dont know how it would impact this particular use case.